terraform/application/databases.tf

```
@@ -29,4 +29,6 @@ module "postgres" {
   use_azure               = var.deploy_azure_backing_services
   azure_enable_monitoring = var.enable_monitoring
   azure_extensions        = ["pg_stat_statements", "pgcrypto"]
+
+  azure_maintenance_window = var.azure_maintenance_window
```

```suggestion
  use_azure                = var.deploy_azure_backing_services
  azure_enable_monitoring  = var.enable_monitoring
  azure_extensions         = ["pg_stat_statements", "pgcrypto"]
  azure_maintenance_window = var.azure_maintenance_window
```

Just one minor thing, could we keep all the azure variables together so they're easier to see as a group.

---

This is fixed .